zOSMF-Samples/workflow_sample_output.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/******************************************************************************/
+/* Copyright Contributors to the zOS-Workflow Project.                        */
+/* SPDX-License-Identifier: Apache-2.0                                        */
+/******************************************************************************/
+-->
+ 
+<workflow xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xsi:noNamespaceSchemaLocation="../workflow/schemas/workflow_v1.xsd">
+ 
+<!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+	 This sample demonstrates the use of load variables from JCL/REXX/Shell script execution.
+	 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+-->
+ 
+<workflowInfo>
+	<workflowID>outputSample</workflowID>
+	<workflowDescription>Sample demonstrating output feature.</workflowDescription>
+	<workflowVersion>1.0</workflowVersion>
+	<vendor>IBM</vendor>
+	<Configuration>
+		<productID>ABC123</productID>
+		<productName>Product ABC</productName>
+		<productVersion>Version 1</productVersion>
+ 	</Configuration>
+</workflowInfo>
+ 
+<!-- Declare a set of variables for use below -->
+<variable name="outputFileNameVariable">
+	<label>File name for output file</label>
+	<abstract>Enter a name for the output properties file</abstract>
+	<description>This value is required. It is included in the JCL for output file path.</description>
+	<category>Output File</category>
+	<string>
+		<default>testOutput</default>
+	</string>
+</variable>
+ 
+<variable name="st_user">
+	<label>user ID</label>
+	<abstract>User ID for the started task.</abstract>
+	<description>The user ID under whose authority the new started task will run.</description>
+	<category>Started</category>
+	<string>
+		<validationType>USERID</validationType>
+		<default>MYSTUSER</default>
+	</string>
+</variable>
+ 
+<variable name="setting0" scope="global">
+	<label>Global string variable</label>
+	<abstract>A global scope string variable.</abstract>
+	<description>This is a global scope string variable for output feature test.</description>
+	<category>Global Variable</category>
+	<string>
+		<default>setting0</default>
+	</string>
+</variable>
+
+<!-- This step is a demo to copy a z/OS UNIX file into an MVS data set. 
+In this sample, dataset "ZOSMF.OUTPUT" is used, the user should create this dataset first or use an existing one.-->  
```

Comment:
```suggestion
In this sample, dataset "ZOSMF.OUTPUT" is used in the output element. The user should create this dataset first or change the value of the output element to an existing data set.-->  
```

Alternatively, it may be easier to use a unix file in /tmp/ for output to avoid having to use datasets.